VideoLocker/res/layout/edx_spinner_item.xml

```diff
@@ -12,4 +12,5 @@
     android:singleLine="true"
     android:textColor="@color/edx_grayscale_neutral_dark"
     android:textSize="@dimen/edx_x_small"
+    android:textDirection="locale"
```

Comment:
> I don't think we should override it's text direction.

However, changing their layout width to `wrap_content` provides the same effect that changing the text direction to `locale` would produce...